include/fp.h

```diff
@@ -0,0 +1,34 @@
+#ifndef _FP_H_
+#define _FP_H_
+#include "ultra64.h"
```

Comment:
```suggestion
#ifndef FP_H
#define FP_H
#include "ultra64.h"
```

Reply:
Oh yeah, we need to do that to all the headers.